File: AOC.Tests/TestUtils.cs
using FluentAssertions;

namespace AOC.Tests;

public static class TestUtils
{
    public static string[] ReadInput(string year, string problem, string suffix)
    {
        return File.ReadAllLines($"../../../../input/{year}/{problem}-{suffix}.txt");
    }

    public static void Test<T>(T expected, Func<string[], T> solver, string[] lines)
    {
        var result = solver(lines);
        result.Should().Be(expected);
    }
}

File: AOC.Python/test_utils.py
def read_input(year, problem, suffix):
    path = f'../input/{year}/{problem}-{suffix}.txt'
    f = open(path, 'r')
    lines = f.readlines()
    lines = [x.strip() for x in lines]

    return lines


def runtest(expected, solver, lines):
    result = solver(lines)
    assert result == expected
